docs/current_docs/integrations/snippets/php/go/main.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"context"
+	"fmt"
+)
+
+type MyModule struct{}
+
+// return container image with application source code and dependencies
+func (m *MyModule) Build(source *Directory) *Container {
+	return dag.Container().
+		From("php:8.2-apache-buster").
+		WithExec([]string{"apt-get", "update"}).
+		WithExec([]string{"apt-get", "install", "--yes", "git-core", "zip", "curl"}).
+		WithExec([]string{"docker-php-ext-install", "pdo", "pdo_mysql", "mysqli"}).
+		WithExec([]string{"sh", "-c", "sed -ri -e 's!/var/www/html!/var/www/public!g' /etc/apache2/sites-available/*.conf"}).
+		WithExec([]string{"sh", "-c", "sed -ri -e 's!/var/www/!/var/www/public!g' /etc/apache2/apache2.conf /etc/apache2/conf-available/*.conf"}).
+		WithExec([]string{"a2enmod", "rewrite"}).
+		WithDirectory("/var/www", source.WithoutDirectory("dagger")).
+		WithWorkdir("/var/www").
+		WithExec([]string{"chown", "-R", "www-data:www-data", "/var/www"}).
```

Comment:
Best set the owner during [`WithDirectory`](https://pkg.go.dev/dagger.io/dagger#ContainerWithDirectoryOpts) rather than with `chown`.

Reply:
Done